docs/uv/metriccallback.rst

```diff
@@ -0,0 +1,88 @@
+Metric Callback
```

Comment:
Nice work on the documentation for this. It takes a while to do this right, but is so important.